packages/asset/contracts/Asset.sol

```diff
@@ -384,5 +384,21 @@ contract Asset is
         return "ASSET";
     }
 
-    uint256[49] private __gap;
+    address private _owner;
```

Comment:
Move to line 59?

Reply:
Yeah I have declared it there so its all close together, but yeah will move it up